frontend/lang/en.json

```diff
@@ -0,0 +1,47 @@
+{
+    "FormField": {
+        "Required": "Required",
+        "Loading": "Loading..."
+    },
+    "LocaleSelector": {
+        "Availabile In": "This site is available in:",
+        "Bottom Right": "You can change the language at any time in the bottom right corner of the page. ↘️"
+    },
+    "ModalError": {
+        "Heading": "An error occurred",
+        "Subheading": "Sorry about that, our team take bug reports seriously, and will try to make it right!",
+        "Error Details": "Error Details", 
```

Comment:
super nit: trailing whitespace, maybe tweak our lint to catch this?

Reply:
Spent 1/2 hr trying to get Json linting working, to no avail. I'm going to punt on this for now.